java/src/main/java/ai/rapids/cudf/Rmm.java

```diff
@@ -119,6 +121,27 @@ public static synchronized void initialize(int allocationMode, LogConf logConf,
     initializeInternal(allocationMode, loc.internalId, path, poolSize);
     MemoryCleaner.setDefaultGpu(Cuda.getDevice());
     initialized = true;
+    Rmm.poolingEnabled = isPool || isArena || isAsync;
+    Rmm.poolSize = poolSize;
+  }
+
+  /**
+   * Get the most recently set pool size or -1 if RMM has not been initialized or pooling is
+   * not enabled.
+   */
+  public static synchronized long getPoolSize() {
+    if (isPoolingEnabled()) {
+      return poolSize;
+    } else {
+      return -1;
+    }
```

Comment:
`poolSize` has default value `-1` right? So should we just return `poolSize`?

Reply:
Not really. If pooling is disabled we would return whatever the pool size is that the user passed in. But it is simple to just clean that up when we set it, so I will do that.